website/addons/badges/model/badges.py

```diff
@@ -161,7 +161,7 @@ def verify(self, vtype='hosted'):
     @property
     def recipient(self):
         return {
-            'idenity': self.node._id,
+            'idenity': self.node._id,  # TODO: An unknown amount of code may depend on this typo
```

Comment:
A quick grep indicates that no code depends on this typo, but still :neutral_face: